src/Tools/dotnet-monitor/ConfigurationJsonWriter.cs

```diff
@@ -20,6 +22,9 @@ namespace Microsoft.Diagnostics.Tools.Monitor
     internal sealed class ConfigurationJsonWriter : IDisposable
     {
         private readonly Utf8JsonWriter _writer;
+        private readonly string _dictionaryName = "IDictionary`2";
```

Comment:
Currently checking via name equality for dictionary/list; do we have a preferred way of doing this?

Reply:
typeof(IDictionary<>).Name